fivox/compartmentLoader.cpp

```diff
@@ -49,8 +49,6 @@ class CompartmentLoader::Impl
         , _report( _config.getReportSource( params.getReport( )),
                    brion::MODE_READ, _target)
     {
-        _report.updateMapping( _target );
-
```

Comment:
updateMapping is implicitly called at construction time:

https://github.com/BlueBrain/Brion/blob/master/brion/plugin/compartmentReportHDF5.cpp#L195
https://github.com/BlueBrain/Brion/blob/master/brion/plugin/compartmentReportCommon.cpp#L37